tests/Fixtures/TestWorkflowListener.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace Tests\Fixtures;
+
+use Illuminate\Contracts\Queue\ShouldQueue;
+
+class TestWorkflowListener implements ShouldQueue
+{
+    public function handle($event)
+    {
+        // NOTE: This doesn't need to do anything as we are just ensuring that the event
+        //       can be serialized and dispatched to a queue.
```

Comment:
nit: I like the comment here, can we add an explicit `return;` just to make it clear from the code side that it's intentionally doing nothing?